src/qtfirebaseadmob.cpp

```diff
@@ -888,6 +888,432 @@ void QtFirebaseAdMobBanner::moveTo(int position)
 
 }
 
+
+/*
+ * AdMobBanner
```

Comment:
I think we should stay close to the SDK naming scheme to avoid future confusion.
So I'll suggest we use "AdMobNativeExpressAd" for this one